src/main/java/htsjdk/tribble/index/tabix/TabixIndex.java

```diff
@@ -230,8 +228,7 @@ public void write(final Path tabixPath) throws IOException {
     @Override
     public void writeBasedOnFeaturePath(final Path featurePath) throws IOException {
```

Comment:
Can you update the javadoc for this method in the `Index` interface to mention that this method now throws an `IOException` in the case where `featurePath` is not a regular file?